tests/exchange/test_gateio.py

```diff
@@ -26,3 +29,48 @@ def test_validate_order_types_gateio(default_conf, mocker):
     with pytest.raises(OperationalException,
                        match=r'Exchange .* does not support market orders.'):
         ExchangeResolver.load_exchange('gateio', default_conf, True)
+
+
+@pytest.mark.parametrize('pair,mm_ratio', [
+    ("ETH/USDT:USDT", 0.005),
+    ("ADA/USDT:USDT", 0.003),
+    # ("DOGE/USDT:USDT", None),
```

Comment:
Is there a reason to have this commented out?
if having `None` as a value is a valid case, i think it should be there (and eventual problems fixed) - otherwise we'll not need this case here?